pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8248.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="Dmitrii Eliseev" id="2022-08-08-file-upload-notification">
+        <addColumn tableName="file_upload">
+            <column name="notification_sent_at" type="datetime(6)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <addColumn tableName="umbrella_study">
+            <column name="notification_email" type="text">
```

Comment:
Did we consider using existing study_email ? Usually study_email is a distribution list that all study staff are on.

Reply:
How is this new column getting populated into the database ? I don't see changes to populate this column as part of study config setup. StudyBuilder::getStudyOrInsert::JdbiUmbrellaStudy.insert().
Singular is already in prod, so probably need patch or update some generic task to handle this update

Reply:
Filling this column with the e-mail is a part of the instruction that Blake prepared for release.
As for the study_email column, I use it as the "from" field. I'm not sure we should have one e-mail address for file uploads and other notifications.